.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*     @googleapis/actools @googleapis/oh-eng @googleapis/yoshi-python
+*     @googleapis/actools @googleapis/oh-eng @googleapis/yoshi-python @googleapis/cloud-client-library-release-automation-team
```

Comment:
Let's remove yoshi-python and oh-eng as well

Reply:
Done.